lib/init.coffee

```diff
@@ -27,22 +16,25 @@ module.exports =
       grammarScopes: @scopes
       scope: 'file'
       lintOnFly: true
-      lint: (textEditor) ->
+      lint: (textEditor) =>
```

Comment:
This binds your function to the last parent scope so when you use `@` it'll be the correct one.